dnsdbq.c

```diff
@@ -2202,21 +2309,28 @@ io_engine(int jobs) {
 	int still, repeats, numfds;
 	struct CURLMsg *cm;
 
-	if (debuglev > 1)
-		fprintf(stderr, "io_engine(%d)\n", jobs);
+	DEBUG(2, (true, "io_engine(%d)\n", jobs));
 
 	/* let libcurl run while there are too many jobs remaining. */
 	still = 0;
 	repeats = 0;
 	while (curl_multi_perform(multi, &still) == CURLM_OK && still > jobs) {
-		if (debuglev > 3)
-			fprintf(stderr, "...waiting (still %d)\n", still);
+		DEBUG(4, (true, "...waiting (still %d)\n", still));
 		numfds = 0;
 		if (curl_multi_wait(multi, NULL, 0, 0, &numfds) != CURLM_OK)
 			break;
 		if (numfds == 0) {
-			if (++repeats > 1)
-				usleep(100000);
+			if (++repeats > 1) {
```

Comment:
Maybe add comment: /* Wait 100ms even if interrupted.  After an interrupt, resume waiting for remaining time. */